### PR TITLE
Added the Table of Contents

### PR DESCRIPTION
**Pull request by wonntann**
Linked to the github blog post for the png image to visualize the generated table of contents from the headings

### Why:

Closes [issue link]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@wonntann 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ To create a heading, add one to six <kbd>#</kbd> symbols before your heading tex
 
 ![Rendered H1, H2, and H6 headings](/assets/images/help/writing/headings-rendered.png)
 
+The use of two or more headings automatically generates a table of contents within the header. Each respective heading title will be listed in the table of contents and navigate within the document to the selected section. 
+
+![Headings Table of Contents](https://i2.wp.com/user-images.githubusercontent.com/7900087/113821370-df915480-9730-11eb-8aed-bdc50e2212d5.gif?ssl=1)
```
Comment (felicitymay) [inline content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:29]: This image isn't showing up in the [preview](https://docs-15218--patch-2.herokuapp.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings). We're also trying stop avoid using animated GIFs in the help as they are bad for accessibility. 

Can I suggest that you add the image below to the `images` directory [here](https://github.com/wonntann/docs/upload/patch-2/assets/images/help/repository) as `table-of-contents.png`? Then we can update this to:

```suggestion
![Screenshot highlighting the table of contents icon](/assets/images/help/repository/table-of-contents.png)
```

table-of-contents

**Review by felicitymay — COMMENTED** [review]
@wonntann - thank you so much for persevering and creating a new PR to replace https://github.com/github/docs/pull/13297 💖 

This looks good. I've made one suggestion to bring the text more into line with our current guidelines which I'll merge, and a second suggestion to fix the image.

When you've added the suggested image, we should be ready to merge this PR and publish your changes 🎉

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md (hunk 1 above)
Comment (wonntann) [reply content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:29]: Ok, sounds good. I committed a png image for the toc. I overlooked the accessibility, if you need any assistance with increasing accessibility please let me know. Thanks!

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 2]
```diff
@@ -24,6 +24,11 @@ To create a heading, add one to six <kbd>#</kbd> symbols before your heading tex
 
 ![Rendered H1, H2, and H6 headings](/assets/images/help/writing/headings-rendered.png)
 
+When you use of two or more headings, GitHub automatically generates a table of contents which you can access by clicking {% octicon "list-unordered" aria-label="The unordered list icon" %} within the file header. Each heading title is listed in the table of contents and you can click a title to navigate to the selected section.
```
Comment (felicitymay) [inline content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:27]: Fix a typo that I introduced 🤦🏻‍♀️ 
```suggestion
When you use two or more headings, GitHub automatically generates a table of contents which you can access by clicking {% octicon "list-unordered" aria-label="The unordered list icon" %} within the file header. Each heading title is listed in the table of contents and you can click a title to navigate to the selected section. 
```

**Comment by 4809381693** [comment]
> Linked to the github blog post for the png image to visualize the generated table of contents from the headings
> 
> 
> 
> ### Why:
> 
> Closes [issue link]
> 
> 
> 
> ### What's being changed:
> 
> 
> 
> ### Check off the following:
> 
> - [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
> - [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> 
> - [ ] This pull request impacts the contribution experience
>   - [ ] I have added the 'writer impact' label
>   - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")
> 
> 
>

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 3]
```diff
@@ -24,6 +24,11 @@ To create a heading, add one to six <kbd>#</kbd> symbols before your heading tex
 
 ![Rendered H1, H2, and H6 headings](/assets/images/help/writing/headings-rendered.png)
 
+The use of two or more headings automatically generates a table of contents within the header. Each respective heading title will be listed in the table of contents and navigate within the document to the selected section.
```
[the remark this answers is not in the document]
Comment (Chipobg) [reply content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md]: []()

**Comment by wonntann** [comment]
Sounds great. I have made the edits for the image. Do you mind guiding me on how to update the image without having to do a different pull request?

**Comment by wonntann** [comment]
>   Thanks @wonntann - happy to give a try.


Thanks for the walk through, I am not able to do either of those options, since I didn't work on it locally nor can I clone it at the moment.

I original added the image via the "Make a contribution" link on the github docs page for this topic at https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Comment by felicitymay** [comment]
@wonntann - One last try, I've just created a test PR from a fork so that I ought to be able to give you better instructions.

1. Click on your branch name at the top of the page `wonntann:patch-2`. This should take you to: https://github.com/wonntann/docs/tree/patch-2.
2. Navigate to the folder that contains the image and click the image (I think this might be: https://github.com/wonntann/docs/blob/patch-2/assets/images/help/repository/headers_toc.png).
3. At the top right of the header for the file, you should see a bin icon. Click this to delete the current image.
4. Commit that change.
5. On the next page, you should find yourself in the `/assets/images/help/repository/` with an **Add file** button. You ought to be able to add the cropped image to the folder and commit this change.

If this doesn't work, I'll find an alternative solution. Thanks for your patience 💖

**Comment by wonntann** [comment]
> If this doesn't work, I'll find an alternative solution. Thanks for your patience sparkling_heart
Thank you so much @felicitymay 
Please let me know if there needs to be any changes on my end. Have a great day!

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 4]
```diff
@@ -24,6 +24,11 @@ To create a heading, add one to six <kbd>#</kbd> symbols before your heading tex
 
 ![Rendered H1, H2, and H6 headings](/assets/images/help/writing/headings-rendered.png)
 
+When you use two or more headings, GitHub automatically generates a table of contents which you can access by clicking {% octicon "list-unordered" aria-label="The unordered list icon" %} within the file header. Each heading title is listed in the table of contents and you can click a title to navigate to the selected section. 
+
+![Screenshot highlighting the table of contents icon](/assets/images/help/repository/headers_toc.png)
```
Comment (felicitymay) [inline content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:29]:
```suggestion
![Screenshot highlighting the table of contents icon](/assets/images/help/repository/headings_toc.png)
```

**Comment by felicitymay** [comment]
> Please let me know if there needs to be any changes on my end. Have a great day!

@wonntann - thanks for persevering and fixing the image 💖 

I think this PR is ready to merge, but I'd like to check the preview first, just to confirm that all is well. Unfortunately, there seems to be a problem with the preview. I'm checking with our engineers about this and hope to be able to merge this PR shortly. After that, your changes will be live within around 24 hours.

**Review by felicitymay — APPROVED** [review]

**Comment by felicitymay** [comment]
Thanks to our wonderful engineers the previews are now working correctly and we can merge this PR 🚀 🥳 

It may take up to 24 hours before you see your change in the live docs. Thanks for your contribution @wonntann 💖

**Pull request merged**